packages/preset-uno/src/rules/behaviors.ts

```diff
@@ -27,3 +30,87 @@ export const listStyle: Rule[] = [
     }
   }],
 ]
+
+const colorResolver
+  = (attribute: string, varName: string) =>
+    ([, body]: string[], { theme }: RuleContext<Theme>) => {
+      const data = extractColor(body)
+
+      if (!data)
+        return
+
+      const { opacity, name, no, color } = data
+
+      if (!name)
+        return
+
+      let useColor = color
+
+      if (!color) {
+        if (name === 'transparent') {
+          return {
+            [attribute]: 'transparent',
+          }
+        }
+        else if (name === 'current') {
+          return {
+            [attribute]: 'currentColor',
+          }
+        }
+        useColor = theme.colors?.[name]
+        if (no && useColor && typeof useColor !== 'string')
+          useColor = useColor[no]
+      }
+
+      if (typeof useColor !== 'string')
+        return
+
+      const rgba = hex2rgba(useColor)
+      if (rgba) {
+        const a = opacity ? opacity[0] === '[' ? h.bracket.percent(opacity)! : (parseFloat(opacity) / 100) : rgba[3]
+        if (a != null && !Number.isNaN(a)) {
+          // @ts-expect-error
+          rgba[3] = typeof a === 'string' && !a.includes('%') ? parseFloat(a) : a
+          return {
+            [attribute]: `rgba(${rgba.join(',')})`,
+          }
+        }
+        else {
+          return {
+            [`--un-${varName}-opacity`]: 1,
+            [attribute]: `rgba(${rgba.slice(0, 3).join(',')},var(--un-${varName}-opacity))`,
+          }
+        }
+      }
+    }
+
+export const boxDecorationBreaks: Rule[] = [
+  ['decoration-slice', { 'box-decoration-break': 'slice' }],
+  ['decoration-clone', { 'box-decoration-break': 'clone' }],
+]
+
+export const caretOpacity: Rule[] = [
+  [/^caret-op(?:acity)?-?(.+)$/, ([, d]) => ({ '--un-caret-opacity': h.bracket.percent(d) })],
+]
+
+export const caretColors: Rule[] = [
+  [/^caret-(.+)$/, colorResolver('caret-color', 'caret')],
+]
+
+export const imageRenderings: Rule[] = [
+  ['image-render-auto', { 'image-rendering': 'auto' }],
+  ['image-render-edge', { 'image-rendering': 'crisp-edges' }],
+  ['image-render-pixel', [
+    ['-ms-interpolation-mode', 'nearest-neighbor'],
+    ['image-rendering', '-webkit-optimize-contrast'],
+    ['image-rendering', '-moz-crisp-edges'],
+    ['image-rendering', '-o-pixelated'],
+    ['image-rendering', 'pixelated'],
```

Comment:
maybe autoprefixer can resolve the prefixs?

Reply:
Yes, but that would require an extra layer of postcss and autoprefixer, which I would actually want to avoid by default (but you can still have if you want to).

Reply:
Without prefix it can cover 95.64%

https://caniuse.com/?search=image-rendering